core/include/scipp/core/variable.h

```diff
@@ -938,6 +941,10 @@ SCIPP_CORE_EXPORT Variable concatenate(const VariableConstProxy &a1,
 SCIPP_CORE_EXPORT Variable dot(const Variable &a, const Variable &b);
 SCIPP_CORE_EXPORT Variable filter(const Variable &var, const Variable &filter);
 SCIPP_CORE_EXPORT Variable mean(const VariableConstProxy &var, const Dim dim);
+SCIPP_CORE_EXPORT Variable mean(const VariableConstProxy &var, const Dim dim,
+                                const VariableConstProxy &masks_sums);
+SCIPP_CORE_EXPORT Variable mean(const VariableConstProxy &var, const Dim dim,
+                                const MasksConstProxy &masks);
```

Comment:
Are all of these required in the header?

Reply:
You are right `VariableConstProxy &masks_sums` doesn't need to be there. I have removed it from the header